security-council-rehearsals/step1/README.md

```diff
@@ -0,0 +1,205 @@
+# Rehearsal 1 - Welcome to SuperchainOps
+
+## Objective
+
+This rehearsal is intended simply to ensure that all the signers feel
+confident running the tooling and performing the validations required
+to execute an onchain action.
+
+Once completed, you can be sure that if the task is executed, the WelcomeToSuperchainOps contract will return `"Welcome to SuperchainOps, <name>"` from its [`welcome()`](https://etherscan.io/address/0x0fb11b4164894912f079de62699f4cc5a2271f0c#readContract#F2) method.
+
+The call that will be executed can be found in the `build` function of the `WelcomeToSuperchainOps` template.
+
+Note that no onchain actions will be taking place during this
+signing. You won’t be submitting a transaction and your address
+doesn’t even need to be funded. These are offchain signatures produced
+with your wallet which will be collected by a Facilitator will execute
+the contract, submitting all signatures for its execution.
```

Comment:
The last line doesn't make grammatical sense, it should be something like:

"These are offchain signatures produced with your wallet. A Facilitator will collect them and use them to execute the transaction that you approved"

Reply:
gotcha, c + pasted from the previous rehearsal. Will update.